src/main/fc/core.c

```diff
@@ -580,9 +580,10 @@ void tryArm(void)
         lastArmingDisabledReason = 0;
 
 #ifdef USE_GPS
-        GPS_reset_home_position();
         //beep to indicate arming
         if (featureIsEnabled(FEATURE_GPS)) {
+            GPS_reset_home_position();
+
```

Comment:
We won't want this here.  We do it in gps.c with other things that happen when arming is done.